configs/classification/EfficientNet_b0.yml

```diff
@@ -8,7 +8,7 @@ lr_finder:
   smooth_f: 0.01
   epochs_warmup: 1
   path_to_savefig: 'lr_finder.jpg'
-  max_lr: 0.01
```

Comment:
Very strange replacement. Is the method so unstable?

Reply:
It is not related to stability. Just if we want to use step size = 0.002, 0.009 is the max number that we can reach. In this case, if we set 0.01, Optuna throws a warning.

Anyway, I set the step size equals to 0.001 and the upper bound to 0.01